packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -2158,6 +2161,13 @@ function flushPassiveEffectsImpl() {
   nestedPassiveUpdateCount =
     rootWithPendingPassiveEffects === null ? 0 : nestedPassiveUpdateCount + 1;
 
+  onPostCommitRootDevTools(root);
+  if (enableProfilerTimer && enableProfilerCommitHooks) {
```

Comment:
Could this go into the HostRoot block of `commitPassiveMountOnFiber`? I think we'd need to re-land #19862 (got reverted when we were bisecting the effects refactor last half)

Reply:
Yeah, we've already got another follow up about that for the Profiler API itself a few lines up (the call to `commitPassiveEffectDurations`). We can just do them both at the same time.